DTAdmin/Groups/SpecialityForGroupViewController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  SpecialityViewController.swift
+//  DTAdmin
+//
+//  Created by Admin on 21.10.17.
+//  Copyright © 2017 if-ios-077. All rights reserved.
+//
+
+import UIKit
+
+class SpecialityForGroupViewController: UIViewController {
+    
+    @IBOutlet weak var specialityTableView: UITableView!
+    
+    var selectSpeciality: ((Speciality) -> ())?
+    var specialities = [Speciality]() {
```

Comment:
Please write SwiftDoc to explain that table will reload when a client set the property.